### ISSUE TITLE
Feature request:  Change listing price

### ISSUE THREAD

**Issue by baedrik** [SecretChainGirl-Foundation/stashh-QA#192]
Multiple people in the discord have requested the ability to change a listing price.  This is already a listing contract function that can only be called by the listing creator called change_price_or_closing_time

**Comment by scarylooking** [comment]
This can already be done by cancelling the listing, setting a new price and then re-listing. I assume you're talking about a way to do this without the cancellation step?

**Comment by baedrik** [comment]
> This can already be done by cancelling the listing, setting a new price and then re-listing. I assume you're talking about a way to do this without the cancellation step?

Yes.  Instead of doing one tx to close a listing and another tx to create new one.  You can do a single tx on the existing listing called `change_price_or_closing_time`, which is significantly cheaper in gas than either of those other two txs just by themselves
